### PR TITLE
Adding function to remove stale objects from kafka logger

### PR DESCRIPTION
**Pull request by sshniro**
Fixes: #1494

### CODE REVIEW

**Comment by membphis** [comment]
@sshniro can we remove the stale object at the end of this function? 

https://github.com/apache/incubator-apisix/pull/1526/files#diff-955d30664be823b32eba60124bd52ae1R136

**Comment by sshniro** [comment]
@membphis I did not understand the last statement, Are you referring to call the function at the end of _M.log function?

**Comment by sshniro** [comment]
@membphis any update? :)

File: apisix/plugins/kafka-logger.lua [hunk 1]
```diff
@@ -21,7 +21,11 @@ local batch_processor = require("apisix.utils.batch-processor")
 local pairs    = pairs
 local type     = type
 local table    = table
+local ipairs   = ipairs
 local plugin_name = "kafka-logger"
+local stale_timer_running = false;
```
Comment (membphis) [inline apisix/plugins/kafka-logger.lua:26]: style: `;` is useless, we should remove it

we can fix it in a new PR

**Pull request merged**